docs/en/stock-management/stock.settings.md

```diff
@@ -0,0 +1,66 @@
+&raquo; [Home](../index.md) / [Inventory Management](./index.md) / Stock Settings
+
+# Stock Settings
+
+There a number of settings that affect how the BHIMA software deals with stock
+management.  To check and update the settings, one goes to the "Stock
+Settings" page (under the "Stock" menu):
+
+![Stock Settings Menu](./images/stock-settings-menu.png)
+
+If you do not see the "Stock Settings" menu item, consult your system
+adminstrator.  You need to be given permission to access that page.
+
+- To add permission for a user to access the "Stock Settings' page: Under
```

Comment:
We probably need documentation for our users and permissions and roles, then we can just include a link to that documentation here.   Ideally, users won't need to have a detailed explanation of the permissions for each module they need access to.

Reply:
Right.  I included this here since the Stock Settings page is a new addition and some folks will inevitably wonder why they do not see it!